src/test/java/com/wikia/webdriver/common/driverprovider/NewDriverProvider.java

```diff
@@ -127,55 +125,52 @@ private static EventFiringWebDriver getAndroidInstance() {
   private static EventFiringWebDriver getFFInstance() {
     // Windows 8 requires to set webdriver.firefox.bin system variable
     // to path where executive file of FF is placed
-    if ("WINDOWS 8".equals(System.getProperty("os.name").toUpperCase())) {
+    if ("WINDOWS 8".equalsIgnoreCase(System.getProperty("os.name"))) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "os.name" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)